src/Commands/Configure.ts

```diff
@@ -82,8 +81,7 @@ export class Configure implements CommandModule {
       };
 
       process.on('SIGTERM', stop).on('SIGINT', stop).on('SIGHUP', stop);
-      console.log(args.networkTestOnly);
-      await app.start({networkTestOnly: !!args.networkTestOnly});
+      await app.start({ networkTestOnly: !!args['network-only'] });
```

Comment:
`yargs` will do aliases automatically, which you can use to access to args) 

```suggestion
      await app.start({ networkTestOnly: !!args.networkOnly });
```